src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/WorkspaceProjectStateChangeDetector.cs

```diff
@@ -70,17 +70,19 @@ public void Dispose()
         _disposeTokenSource.Dispose();
     }
 
-    private async ValueTask ProcessBatchAsync(ImmutableArray<(Project? Project, IProjectSnapshot ProjectSnapshot)> items, CancellationToken token)
+    private ValueTask ProcessBatchAsync(ImmutableArray<(Project? Project, IProjectSnapshot ProjectSnapshot)> items, CancellationToken token)
```

Comment:
should this just be synchronous now?

Reply:
It can't be. `AsyncBatchingWorkQueue` takes a func that returns `ValueTask`.

Reply:
ah gotcha, didn't see that.